src/ngraph/op/fused/conv_fused.cpp

```diff
@@ -220,7 +297,6 @@ op::ConvolutionBiasBackpropFiltersBias::ConvolutionBiasBackpropFiltersBias(
             "match");
```

Comment:
The check for element type mismatch should be done in `validate_and_infer_types()`, not in the constructor.

Reply:
Will remove it since its redundant. convbprop_filters will check for this.